content/pt-br/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Configurar o acesso a múltiplos clusters
+content_type: task
+weight: 30
+card:
+  name: tasks
+  weight: 25
+  title: Configure access to clusters
+---
+
+<!-- overview -->
+
+Esta página mostra como configurar o acesso a vários clusters usando arquivos de configuração. Depois que os clusters, os usuários e os contextos forem definidos em um ou mais arquivos de configuração, você pode alternar rapidamente entre os clusters usando o comando `kubectl config use-context`.
+
+{{< note >}}
+Um arquivo usado para configurar o acesso a um cluster às vezes é chamado de arquivo *kubeconfig*. Essa é uma forma genérica de se referir a arquivos de configuração. Isso não significa que exista um arquivo chamado `kubeconfig`.
+{{< /note >}}
+
+
+{{< warning >}}
+Use somente arquivos kubeconfig de fontes confiáveis. O uso de um arquivo kubeconfig especialmente criado, pode resultar em execução de código malicioso ou exposição de arquivos. Se você preciso usar um arquivo kubeconfig não confiável, inspecione-o cuidadosamente antes, da mesma forma que faria com um script de shell.
+{{< /warning>}}
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Para verificar se {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} está instalado,
+execute `kubectl version --client`. A versão do kubectl deve ser [dentro de uma versão secundária](/releases/version-skew-policy/#kubectl) do servidor de API do seu cluster.
```

Comment:
We currently use "maior"/"menor"/"correção" to define major/minor/patch versions of Kubernetes in the [versions page](https://kubernetes.io/pt-br/releases/). This is just a suggestion to make it clearer to the reader, but feel free to suggest a different way of stating this if you feel it could be better:

```suggestion
Para verificar se {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} está instalado,
execute `kubectl version --client`. A versão do kubectl deve ter [no máximo uma versão menor de diferença](/releases/version-skew-policy/#kubectl) da versão do servidor de API do seu cluster.
```

Reply:
That does sound much better, and I've noted it for future localizations.

Reply:
That does sound much better, and I've noted it for future localizations.